agent/ecscni/plugin.go

```diff
@@ -96,6 +99,8 @@ func (client *cniClient) CleanupNS(cfg *Config) error {
 	}
 
 	seelog.Debugf("Starting clean up the container namespace: %s", cfg.ContainerID)
+	os.Setenv("ECS_CNI_LOGLEVEL", logger.GetLevel())
+	defer os.Unsetenv("ECS_CNI_LOGLEVEL")
```

Comment:
I was thinking can we set all the environment variable needed by the plugin at the agent start, when we check if this feature is [enabled](https://github.com/aws/amazon-ecs-agent/blob/enis/agent/app/agent.go#L194)?